core/src/main/java/com/linecorp/armeria/client/websocket/WebSocketClientBuilder.java

```diff
@@ -80,6 +81,7 @@ public final class WebSocketClientBuilder extends AbstractWebClientBuilder {
     private int maxFramePayloadLength = DEFAULT_MAX_FRAME_PAYLOAD_LENGTH;
     private boolean allowMaskMismatch;
     private List<String> subprotocols = ImmutableList.of();
+    private boolean aggregateContinuation;
```

Comment:
minor question) Maybe it is just me, but I feel like most users won't be interested in continuation frames and `true` would be the more sensible default.

Reply:
I think we may provide a higher API that can automatically aggregate continuation frames with `StreamMessage<String>` or `StreamMessage<ByteBuf>`.

Reply:
> most users won't be interested in continuation frame

I completely agree with your perspective on this matter, but I believe that setting true for `aggregateContinuation` as a default is a separate consideration.
If continuation frames are used that the server developers do not notice, the server may experience memory pressure. So I prefer to set it false. 🤔

Reply:
> I think we may provide a higher API that can automatically aggregate continuation frames with StreamMessage<String> or StreamMessage<(ByteBuf|byte[])>.

Could you share an example of the usage? I'm not sure how it's going to be used because what we need here is the stream message of `WebSocketFrame` instead of strings or byte arrays. 🤔

Reply:
Continuation is a low-level detail that many users might not be interested in for their business logic. I think we can abstract a set of WebSocket frames followed by continuation frames with `StreamMessage` so that users easily aggregate the whole message or subscribe to the messages one by one.

```java
TextStreamMessage extends StreamMessage<String> {
   CompletableFuture<String> aggregate();
}

ByteStreamMessage extends StreamMessage<byte[]> {
   CompletableFuture<byte[]> aggregate();
}

interface WebSocketStreamHandler {
   void onText(RequestContext ctx, TextStreamMessage texts, WebSocketWriter out);
   
   void onBinary(RequestContext ctx, TextStreamMessage bytes, WebSocketWriter out);
   ...  
}
```

```java
class EchoWebSocketStreamHandler implements WebSocketStreamHandler {
   void onText(RequestContext ctx, TextStreamMessage texts, WebSocketWriter out) {
     // If users want to get the whole text
     texts.aggregate().thenAccept(text -> {
        out.write(text);
     });
     // To get a text frame one by one with backpressure
     texts.subscribe(...)
   }
   
   void onBinary(RequestContext ctx, TextStreamMessage bytes, WebSocketWriter out);
}
```

Reply:
Agreed. We don't know which one is preferred by users. Both versions could be provided later if necessary.